src/playbook.yaml

```diff
@@ -3,22 +3,35 @@
 
 - hosts: all
   become: yes
-
   tasks:
     - name: Upgrade all packages
-      yum: name=* state=latest
+      yum:
+        name: "*"
+        state: latest
+        update_cache: true
 
     - name: Install system packages
       ansible.builtin.yum:
         name: "{{ item }}"
         state: present
       with_items:
-        - "curl"
         - "wget"
-        - "jq"
         - "git"
-        - "amazon-linux-extras"
-        - "docker"
+        - "python3-pip"
+        - "python3.11-pip"
```

Comment:
what's the rational to installing both of these?

Reply:
Hm, I thought I saw a request specifically for 3.11, and according to https://docs.aws.amazon.com/linux/al2023/ug/python.html we would still need 3.9; but I can't find that request for 3.11 anywhere in jira or slack so I'm starting to think I might've imagined it.